sonar/affiliations/affiliation_resolver.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+#
+# Swiss Open Access Repository
+# Copyright (C) 2019 RERO
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, version 3 of the License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Affiliations resolver."""
+
+import csv
+
+from fuzzywuzzy import fuzz
+from werkzeug.utils import cached_property
+
+CSV_FILE = './data/affiliations.csv'
+
+
+class AffiliationResolver():
```

Comment:
The name Resolver can be confusing with the invenio PID resolver.

Reply:
I have no other idea, I will keep this name